benefits/core/templates/core/includes/lang-selector.html

```diff
@@ -14,8 +14,9 @@
     <form method="post" action="{% url "set_language" %}">
       {% csrf_token %}
       {% get_language_info for code as langinfo %}
+      <input name="next" type="hidden" value="{{ redirect_to }}" />
       <input name="language" type="hidden" value="{{ code }}" />
-      <button class="btn btn-lg btn-outline-light" type="submit">{{ langinfo.name_local | capfirst }}</button>
+      <input class="btn btn-lg btn-outline-light" type="submit" value="{{ langinfo.name_local | capfirst }}">
```

Comment:
Just wanted to confirm: this change wasn't actually needed to fix the bug, right? That is, there's no behavioral difference, and it's just a different way of defining the same submit button.

Reply:
Thanks @angela-tran , that's a good point! Changing `button` to `input` is actually not needed to fix the bug so I'll revert that change by pushing one more time. I think doing this keeps things cleaner.